straxen/plugins/nveto_recorder.py

```diff
@@ -66,6 +73,19 @@ def setup(self):
             self.baseline_samples = straxen.get_correction_from_cmt(
                 self.run_id, self.config['baseline_samples_nv'])
 
+        # Check config of `hit_min_amplitude_nv` and define hit thresholds
+        # if cmt config
+        if is_cmt_option(self.config['hit_min_amplitude_nv']):
+            test_is_cmt_option()
```

Comment:
Just to make sure @FaroutYLq, Jorans comment addresses only the `test_is_cmt_option` part. Sorry, I was not very clear hear in the beginning. `test_is_cmt_option` should run together with our other automatic tests to catch if someone changes the structure of CMT options but forgets to updated the other option configs.

Reply:
Thanks Daniel for the clarification. I am a bit confused with `test_cmt.py`: where are the functions in this script called? I am not sure if the `test_is_cmt_option` will be called if I just simply move it to `test_cmt.py`...

Reply:
@FaroutYLq, yes it will, it's whats called in the automated tests on github. To trigger them locally, you can do `cd straxen ; pytest` (assuming pytest installed)